openquakeplatform/openquakeplatform/templates/documents/document_detail.html

```diff
@@ -24,7 +24,13 @@
 
 {% block body_outer %}
   <div class="span5">
-    <h2 class="page-title">{{ document.title }}</h2>
+    <h2 class="page-title">
+      {% if document.owner.username == 'GEM' %}
+        {{ document.title }}
+      {% else %}
+        {{ document.title }} (User Contributed)
+      {% endif %}
```

Comment:
This can be done in a more simple way:
```python
{{ document.title }}
{% if document.owner.username == 'GEM' %}
  (User Contributed)
{% endif %}
```